web-common/src/features/dashboards/url-state/convertURLToExplorePreset.ts

```diff
@@ -151,6 +151,29 @@ export function convertURLToExplorePreset(
     }
   }
 
+  if (searchParams.has(ExploreStateURLParams.SortType)) {
```

Comment:
Why do we need to copy this piece of code? `fromExploreUrlParams` should be enough no?